Assets/Scripts/Chest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Chest : MonoBehaviour

```

Comment:
The chest looks awesome! I love it.

I think that in next days we can discuss how we could split that script into separate behaviors in a clear way.